### PR TITLE
[new release] omd (2.0.0.alpha3)

### PR DESCRIPTION
**Pull request by shonfeder**
A Markdown frontend in pure OCaml

- Project page: <a href="https://github.com/ocaml/omd">https://github.com/ocaml/omd</a>

##### CHANGES:

- Expose the HTML escape function `htmlentities` (ocaml/omd#295 @cuihtlauac)

- Support generation of identifiers in headers (ocaml/omd#294, @tatchi)

- Support GitHub-Flavoured Markdown tables (ocaml/omd#292, @bobatkey)

- Update parser to support CommonMark Spec 0.30 (ocaml/omd#266, @SquidDev)

- Preserve the order of input files in the HTML output to stdout (ocaml/omd#258,
  @patricoferris)

- Fix all deviations from CommonMark Spec 0.30 (ocaml/omd#284, ocaml/omd#283, ocaml/omd#278, ocaml/omd#277, ocaml/omd#269,
  @tatchi)

### CODE REVIEW

File: packages/omd/omd.2.0.0.alpha3/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "Nicolás Ojeda Bär <n.oje.bar@gmail.com>"
+]
+license: "ISC"
+tags: ["org:ocamllabs" "org:mirage"]
+homepage: "https://github.com/ocaml/omd"
+bug-reports: "https://github.com/ocaml/omd/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "stdcompat"
```
Comment (mseri) [inline packages/omd/omd.2.0.0.alpha3/opam:24]:
```suggestion
  "stdcompat" {>= "16"}
```

uses String.for_all from 4.13 which is introduced in this version (and List.find_map from 4.10.0)

**Comment by mseri** [comment]
Looks good, thanks

**Comment by kit-ty-kate** [comment]
The version number is not correct. This should be `2.0.0~alpha3` not `2.0.0.alpha3`. I'm reverting. Could you reopen a PR with the correct version number?

**Comment by mseri** [comment]
Well spotted. I sent the PR

**Pull request merged**